app/javascript/packages/document-capture/components/location-collection-item.scss

```diff
@@ -12,4 +12,9 @@
   margin-top: 1rem;
   padding-bottom: 1rem;
   border-color: color('primary-light');
+
+  &:last-child {
+    border-bottom-style: none;
+    padding-bottom: 0;
+  }
```

Comment:
From a design system perspective, the utility can be good to use [tokens](https://designsystem.digital.gov/design-tokens/) over hard-coded values. But since `0` is `0`, I wouldn't think it's a big deal either way.

Reply:
@allthesignals  It looks like for reusability purposes and if/when passing in a variable might will help determine the style. https://stackoverflow.com/questions/45194144/why-use-mixin-of-padding-and-margin-in-sass